cli/compose/template/template_test.go

```diff
@@ -220,7 +220,7 @@ func TestExtractVariables(t *testing.T) {
 		{
 			name: "required-variable",
 			dict: map[string]any{
-				"foo": "${bar?:foo}",
+				"foo": "${bar:?foo}",
```

Comment:
Looks like a typo imo, I think the standard parameter expansion syntax uses :?, which is also what is used in all the other assertions in this file. Even if it is not a typo, seems like it would still be best to keep it consistent.

Reply:
i'm not so sure, the outcome seems to still be the same. I might be wrong here, but from a quick glance it seems any of these `?, :?, ?:` would match here: https://github.com/docker/cli/blob/master/cli/compose/template/template.go#L166-L169 

I would just revert this change

Reply:
I reverted the change :)